spec/models/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe User do
+
+  let!(:user) do
+    User.create(email: 'test@test.com', password: 'secret1234',
+               password_confirmation: 'secret1234', name: 'test', username: 'testest')
```

Comment:
Line is too long. [86/80]<br>Align the elements of a hash literal if they span more than one line.